src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.wrapper {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  min-width: 100px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.search {
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300px;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+  height: 70px;
+  padding-left: 62px;
+  color: #3d4e61;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: "Avenir-Heavy", sans-serif;
+  padding-left: 62px;
+  outline: none;
+}
+
+.search-icon2 {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search--small {
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+  padding-left: 33px;
+  line-height: 20px;
+}
+
+.search--small:focus {
+  padding-left: 33px;
+}
```

Comment:
in focus state, as per figma design placeholder should not change font-weight but what ever form input we write that should be bold 
but in your case when we click on input field place holder also becomes bold 
so please consider the last point in the checklist
`Remember that placeholder has its own set of styles available using ::placeholder pseudo-element.`